src/pages/instances/EditInstance.tsx

```diff
@@ -253,19 +257,15 @@ const EditInstance: FC<Props> = ({ instance }) => {
       </Form>
       <FormFooterLayout>
         {readOnly ? (
-          <Button
-            appearance="positive"
-            onClick={() => {
-              void formik.setFieldValue("readOnly", false);
-            }}
-          >
-            Edit instance
-          </Button>
+          <div style={{ height: "3.5rem" }} />
```

Comment:
what if we add `min-height: 3.5rem` to the `bottom-btns` class used by the `footer` element in `FormFooterLayout` component instead of adding this `div` here? Applicable for all the other `Edit...` components.

Reply:
I was thinking about this, we might want to keep the "Edit ..." buttons here actually instead of removing them.

Reply:
Actually a great idea to use the min-height instead, applied that change now.